Lib/collections/__init__.py

```diff
@@ -348,12 +349,33 @@ def __getnewargs__(self):
 
 {field_defs}
 """
-
-_repr_template = '{name}=%r'
-
 _field_template = '''\
     {name} = _property(_itemgetter({index:d}), doc='Alias for field number {index:d}')
 '''
+_repr_template = '{name}=%r'
+_new_template = '''
+def __new__(_cls, {arg_list}):
+    'Create new instance of {typename}({arg_list})'
+    return _tuple.__new__(_cls, ({arg_list}))
```

Comment:
Instead of `_tuple.__new__`, how about put `namespace['_tuple_new'] = tuple.__new__` and call `_tuple_new`?
It makes `eval` and instantiation bit (about 3%) faster.

this pull request:
```
$ ./pyenv/versions/3.6.2/bin/python -m perf timeit -s 'from collections import namedtuple' -- 'namedtuple("Foo", "foo bar baz")'
.....................
Mean +- std dev: 90.1 us +- 3.3 us

$ ./pyenv/versions/3.6.2/bin/python -m perf timeit -s 'from collections import namedtuple; Foo=namedtuple("Foo", "foo bar baz")' -- 'Foo(1,2,3)'
.....................
Mean +- std dev: 503 ns +- 15 ns
```

`_tuple_new` version:

```
$ ./pyenv/versions/3.6.2/bin/python -m perf timeit -s 'from collections import namedtuple' -- 'namedtuple("Foo", "foo bar baz")'
.....................
Mean +- std dev: 87.1 us +- 2.7 us

$ ./pyenv/versions/3.6.2/bin/python -m perf timeit -s 'from collections import namedtuple; Foo=namedtuple("Foo", "foo bar baz")' -- 'Foo(1,2,3)'
.....................
Mean +- std dev: 485 ns +- 21 ns
```

Reply:
Thanks, making that change.